src/actions/ExpandableText.tsx

```diff
@@ -56,7 +61,7 @@ const ExpandableText = (props: ExpandableTextProps) => {
     button = moreLessButton(expanded, setExpanded, props.strings, props.buttonClassName)
   }
   return (
-    <div className={`expandable-text ${props?.className}`}>
+    <div className={`expandable-text ${props?.className ? props.className : ""}`}>
```

Comment:
I kept getting errors when I tried to push until I made this change. Is this the correct approach?

Reply:
This works and you don't need to change what you have. But you can do `${props?.className || ""}`

Reply:
I believe we do get errors this way for showing "undefined" - not sure if that's what you were seeing Chad?

Reply:
The specific error that I get is `Invalid type "string | undefined" of template literal expression.`

@ludtkemorgan `${props?.className || ""}` works, I will change that over so that we're more concise.